hack/loki-distributed.sh

```diff
@@ -44,6 +44,10 @@ kubectl apply -n $NAMESPACE -f examples/distributed-loki/3-services/
 echo ""
 echo "Deployment complete"
 echo ""
+echo "Configure FlowCollector Loki with:"
+echo "    url: 'http://loki-distributed-distributor.netobserv.svc.cluster.local:3100/'"
+echo "    querierUrl: 'http://loki-distributed-query-frontend.netobserv.svc.cluster.local:3100/'"
```

Comment:
I think, the namespace here should be network-observability here

Reply:
the default namespace changed: https://github.com/netobserv/documents/pull/26/files#diff-d828b4887a8f430cebe46d5cc46bed8578de7639942a7e66bb6bfa5ad2b55220

Reply:
well, actually it should be `${NAMESPACE}`

Reply:
(done)

Reply:
Missed this changed, sorry.